app-interface/cronjobs/rhose-stage-prolong.yml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: template.openshift.io/v1
```

Comment:
Is this job deployed also in prod? We only need it in stage. I don't understand if this is going to apply everywhere